src/controllers/update/update.manage.trusts.tell.us.about.the.former.bo.controller.ts

```diff
@@ -0,0 +1,90 @@
+import { NextFunction, Request, Response } from 'express';
+
+import { logger } from '../../utils/logger';
+import {
+  ROUTE_PARAM_TRUSTEE_ID,
+  UPDATE_MANAGE_TRUSTS_REVIEW_FORMER_BO_URL,
+  UPDATE_MANAGE_TRUSTS_TELL_US_ABOUT_THE_FORMER_BO_PAGE,
+} from '../../config';
+import { getApplicationData, setExtraData } from '../../utils/application.data';
+import { Session } from '@companieshouse/node-session-handler';
+import { Trust, TrustHistoricalBeneficialOwner } from '../../model/trust.model';
+import { TrusteeType } from '../../model/trustee.type.model';
+import { TrustHistoricalBeneficialOwnerForm } from '../../model/trust.page.model';
+import { validationResult } from 'express-validator';
+import { FormattedValidationErrors, formatValidationError } from '../../middleware/validation.middleware';
+import { saveAndContinue } from '../../utils/save.and.continue';
+import { mapBeneficialOwnerToSession, mapFormerTrusteeFromSessionToPage } from '../../utils/trust/historical.beneficial.owner.mapper';
+import { getTrustInReview, getTrustee, getTrusteeIndex } from '../../utils/update/review_trusts';
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `${req.method} ${req.route.path}`);
+
+    const appData = getApplicationData(req.session);
+    const trust = getTrustInReview(appData) as Trust;
```

Comment:
As comment above - may be better to redirect or error if no trust in review.

Reply:
Same as above, middleware will catch and redirect if no trust in review